containers/README.md

```diff
@@ -0,0 +1,9 @@
+# Node images have moved
+
+Warewulf node images are now being maintained in a seprate repository:
```

Comment:
s/seprate/separated/

Reply:
I had actually meant for this to be "separate." Fixed. Thanks!